lib/modules/auth/login/login_screen.dart

```diff
@@ -1,232 +1,210 @@
-                                      loginModel: loginModel,
-                                    );
-                                  }
-                                },
-                                child: Text("LOGIN"),
-                              ),
-                            );
-                          },
-                          fallback: (context) => buildLoadingState(),
-                        ),
-                        SizedBox(height: 16),
-                        Row(
-                          mainAxisAlignment: MainAxisAlignment.center,
-                          children: [
-                            Text("Don't have an account?"),
-                            TextButton(
-                                onPressed: () =>
-                                    navigateToRegisterScreen(context),
-                                child: Text(
-                                  "Register",
-                                  style: TextStyle(
-                                      fontWeight: FontWeight.bold,
-                                      fontSize: 14),
-                                )),
-                          ],
-                        )
-                      ],
+    return BlocProvider<LoginBloc>(
+      create: (context) => LoginBloc(LoginRepository()),
+      child: Scaffold(
+        appBar: AppBar(),
+        body: BlocConsumer<LoginBloc, LoginState>(
+          listener: (context, state) {
+            if (state is LoginSuccessState) {
+              buildToastMessage(state.model.message, Colors.green);
+              navigateToHomeScreen(context);
+            }
+            else if (state is LoginErrorState) {
+              buildToastMessage(state.message, Colors.red);
+            }
+            else if (state is LoginEmailAndPasswordValidationState){
+              emailErrorMessage = state.emailMessage;
+              passwordErrorMessage =  state.passwordMessage;
+            }
+          },
+          builder: (context, state) {
+            return SingleChildScrollView(
+              child: Padding(
+                padding: const EdgeInsets.all(16.0),
+                child: Column(
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: [
+                    Text(
+                      "Login",
+                      style: Theme.of(context).textTheme.headline4.copyWith(
+                          fontWeight: FontWeight.bold, color: Colors.black),
                     ),
-                  ),
+                    Text("Login now to browse our hot offers"),
+                    SizedBox(height: 40),
+                    buildEmailTextField(),
+                    buildSizedBoxSeperator(),
+                    buildPasswordTextField(context),
+                    buildSizedBoxSeperator(),
+                    buildSignInButton(state),
+                    buildSizedBoxSeperator(),
+                    buildDonotHaveAnAccountButton(context)
```

Comment:
delete build and every widget must end with and widget ex (emailTextFieldWidget)

Reply:
done